Makefile

```diff
@@ -111,7 +111,7 @@ DOCKER_IMAGE?=$(PROJECT_NAME)
 DOCKER_IMAGE_TAG?=$(COMMIT)
 export DOCKER_IMAGE_TAG
 DOCKER_REGISTRY?=docker.io
-DOCKER_REPOSITORY?=aslakknutsen
+DOCKER_REPOSITORY?=maistra
```

Comment:
@aslakknutsen I guess this we wanted to keep in the old account before we sort out access to docker hub?

Reply:
ah right!

Reply:
https://github.com/Maistra/istio-workspace/pull/111/commits/7c97fbf255f0e77e123360580ce257d2de71234e